document-store/src/main/java/org/hypertrace/core/documentstore/mongo/MongoCollection.java

```diff
@@ -409,10 +408,13 @@ public boolean deleteAll() {
     return true;
   }
 
-  private void parseOrderByQuery(List<OrderBy> orderBys, Map<String, Object> orderbyMap) {
+  @VisibleForTesting
+  LinkedHashMap<String, Object> parseOrderByQuery(List<OrderBy> orderBys) {
+    LinkedHashMap<String, Object> orderByMap = new LinkedHashMap<>();
     for (OrderBy orderBy : orderBys) {
-      orderbyMap.put(orderBy.getField(), (orderBy.isAsc() ? 1 : -1));
+      orderByMap.put(orderBy.getField(), (orderBy.isAsc() ? 1 : -1));
```

Comment:
Can you extend the test `testOffsetAndLimitOrderBy` with multiple orderBy in [DocStoreTest.java](https://github.com/hypertrace/document-store/blob/081cf1b5f9433e22d1a8736d01e1cc85dd663109/document-store/src/integrationTest/java/org/hypertrace/core/documentstore/DocStoreTest.java#L827), so, it can verify Postgres side too. However, I have checked the Postgres side impl, it looks fine for multiple orderby.

Reply:
Sure, extended that test to add multiple orderBys.
Also, added a unit test for `parseOrderBys` method for Postgres.